backend/app/api/endpoints/admin/kind_management.py

```
@@ -0,0 +1,349 @@
+@router.delete(
+    "/users/{user_id}/kinds/{kinds}/{name}", status_code=status.HTTP_204_NO_CONTENT
+)
+async def admin_delete_user_resource(
+    user_id: int = Path(..., description="User ID"),
+    kinds: str = Path(
+        ...,
+        description="Resource type. Valid options: ghosts, models, shells, bots, teams, workspaces, tasks",
+    ),
+    name: str = Path(..., description="Resource name"),
+    namespace: str = Query("default", description="Resource namespace"),
+    db: Session = Depends(get_db),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)